admin_test.go

```diff
@@ -0,0 +1,39 @@
+package gapi
+
+import (
+	"testing"
+)
+
+const (
+  createUserJSON = `{"id":1,"message":"User created"}`
```

Comment:
go fmt?

Reply:
Weird, something must be configured incorrectly with my editor, I'll run it now.